modules/rostests/apitests/shell32/InternalExtractIconListW.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Tests for InternalExtractIconListW
+ * COPYRIGHT:   Copyright 2025 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#include "shelltest.h"
+#include <undocshell.h>
+#include <versionhelpers.h>
+
+START_TEST(InternalExtractIconListW)
+{
+    if (IsWindowsVistaOrGreater())
+    {
+        skip("InternalExtractIconListW of Vista+ is useless\n");
```

Comment:
in what sense it is useless?

Reply:
The symptom of Vista+ is like below:

![Win10](https://github.com/user-attachments/assets/4218decd-b67e-4070-bb4e-c4050cfba3c1)

Reply:
it looks like (at least on win10) it because a stub.

Reply:
You may actually `ok(hPairs == NULL, ...);`,
and, as it seems trivial, you might implement that Vista+ behavior too.

Reply:
A test added.